### DISCUSSION TITLE
[Feature] Don't Sync Videos on Cellular

### DISCUSSION THREAD

**Discussion by aarhor** [immich-app/immich#8622] [Feature Request]
### The feature

I'm using Immich now for some weeks and already love it, but i'm missing a very important feature from Google Photos. It's the option to choose if i want to sync Videos and or Photos when the mobile data is turned on. So when i made an 4K Video which has 2 GB, that would mostly crash my data volume...

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #8463
